lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -4937,6 +4937,32 @@ struct FDivOpConversion
                      ConversionPatternRewriter &rewriter, Type elemTy,
                      ValueRange operands, Location loc) const {
 
+#ifdef USE_ROCM
+    const std::string readConstraint = "v";
+    const std::string writeConstraint = "=v";
+
+    GCNBuilder gcnBuilder;
+    unsigned bitwidth = elemTy.getIntOrFloatBitWidth();
+
+    auto res = gcnBuilder.newOperand(writeConstraint);
+    auto lhs = gcnBuilder.newOperand(operands[0], readConstraint);
+    auto rhs = gcnBuilder.newOperand(operands[1], readConstraint);
+
+    // create inst
+    auto &rcp = gcnBuilder.create<GCNInstr>("v_rcp")->float_op_type(bitwidth);
+    auto &mul_inst = gcnBuilder.create<GCNInstr>("v_mul")->float_op_type(bitwidth);
+    // auto &div_inst = gcnBuilder.create<GCNInstr>("v_div_fixup")->float_op_type(bitwidth); // TODO: fix mismatches
```

Comment:
codes need a cleanup

Reply:
I wanted to keep the TODO and code for a subsquent PRs. @rsanthanam-amd, do you have issue with this?

Reply:
i would suggest removing this comment to keep the code clean.

Reply:
Perhaps keep the TODOs here in the description of the PR.

Reply:
Done